packages/@react-spectrum/sidenav/test/SideNav.test.js

```diff
@@ -83,13 +83,19 @@ function renderComponent(Name, Component, ComponentSection, ComponentItem, props
 
 describe('SideNav', function () {
   let stub1, stub2;
+  let scrollHeight;
+
   beforeAll(function () {
+    jest.useFakeTimers();
+    scrollHeight = jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 48);
```

Comment:
if scrollHeight isn't mocked, we get an extraneous state update that we don't want